library/src/cache.rs

```diff
@@ -159,7 +159,7 @@ impl UpdaterState {
     /// - The updater has been initialized but no boot recorded yet.
     pub fn current_boot_patch(&self) -> Option<PatchInfo> {
         if let Some(slot_index) = self.current_boot_slot_index {
-            return self.patch_info_at(slot_index)
+            return self.patch_info_at(slot_index);
```

Comment:
For my own understanding, why add a semicolon here?

Reply:
It's optional I believe. I think this was just I had rust format-on-save off by accident and finally fixed that and thus it formatted a few files.

When you have:
{
  foo
}

that returns the value "foo" from the block, e.g:

let bar = {
   foo
};

that's why you often see function end with:

Ok(())

Which returns an Ok (a type of Result, wrapping the void value () ).